src/test/java/uk/gov/companieshouse/confirmationstatementapi/service/ConfirmationStatementServiceTest.java

```diff
@@ -246,6 +248,59 @@ void getConfirmationSubmissionNotFound() {
         assertFalse(result.isPresent());
     }
 
+    @Test
```

Comment:
i'd also add a test in here for when submissionData is null (to cover line 151 in confirmationStatementService)